src/dataflow-types/src/types/sources.proto

```diff
@@ -29,6 +29,11 @@ message ProtoKeyEnvelope {
     }
 }
 
+message ProtoUpsertEnvelope {
+    ProtoUpsertStyle style = 1;
+    repeated uint64 key_indices = 2;
```

Comment:
have we decided, that usize is always mapped to `uint64`? that seems to be the case, see `ProtoKey`,  but do we have this convention written in docs anywhere?

Reply:
I'm not sure it's in the docs, but it's implicitly given in code, since `impl ProtoRepr for usize` does convert to u64.

(I haven't updated myself to the new `RustType` interface, maybe it's called different now)